README.md

```diff
@@ -1,39 +1,53 @@
 
-A plugin for managing your Pantheon secrets via Terminus.
+## Key Features
 
-NOTE: Secrets Manager is still in Early Access. Customer Support is unable to provide assistance with this feature. Please create an Issue in the GitHub repo to report any issues or bugs.
+- Securely host and maintain Secrets on Pantheon
+- Use private repositories in Integrated Composer builds
+- Create and update secrets via Terminus
+- Ability to set a `COMPOSER_AUTH` environment variable and/or a `Composer auth.json` authentication file with Terminus commands
+- Ability to define site-specific Secrets (organization-specific Secrets are not yet supported)
+- Ability to define the level of secrecy for each managed item (this determines which users can view the value of a Secret after entering it)
+- Secrets are encrypted at rest
+
+## Secrets Management Plugin Requirements
+
+Secrets Management requires the following:
+
+- A Pantheon account
+- Integrated Composer
+- Terminus
+
+## Early Access
+
+The Secrets Management plugin is available for Early Access participants. Features for Secrets Management are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. Visit the [Pantheon Slack channel](https://slackin.pantheon.io/) (or sign up for the channel if you don't already have an account) to learn how you can enroll in our Early Access program. Please review [Pantheon's Software Evaluation Licensing Terms](https://legal.pantheon.io/#contract-hkqlbwpxo) for more information about access to our software.
 
 ## Installation
 
-To install this plugin using Terminus 3:
+Terminus 3.x has built in plugin management.
+
+Run the command below to install Terminus Secrets Management.
+
 ```
 terminus self:plugin:install terminus-secrets-manager-plugin
 ```
 
-## Usage
+## Terminus Secrets Management Commands
 
-### Listing secrets
+### Set a Secret
 
-Use `terminus secret:list` to list existing secrets for a given site:
+The Secrets `set` command takes the following format:
 
-```
-terminus secret:list <site>
+- `Name`
+- `Value`
+- `One or more scopes`
 
- ------------- ------------- ---------------------------
-  Secret name   Secret type   Secret value
- ------------- ------------- ---------------------------
-  file.json     file          contents of a secret file
-  foo           env           bar
- ------------- ------------- ---------------------------
-```
+The scope determines access to the Secret’s value. For example, if the scope is set to `users`, it will allow the user to view the Secret in Terminus. If the scope is set to `integrated composer`, it makes the Secret available to the Integrated Composer build. 
 
-### Setting secrets
 
-Use `terminus secret:set <site> <secret_name> <secret_value> [--type=TYPE] [--scope=SCOPE]` to set a secret for a given site:
+Run the command below to set a Secret in Terminus:
 
 ```
 terminus secret:set <site> foo bar
```

Comment:
Rather than foo bar, perhaps we should use <secret-name> and <secret-value> ? @whitneymeredith